Exercise6/Program.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise6
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Enter the first edge: ");
+            var firstEdge = Convert.ToInt32(Console.ReadLine());
```

Comment:
Create a function and call `var firstEdge = ReadInputInt("Enter the first edge: ");`

Reply:
Done